kafka/protocol/admin.py

```diff
@@ -333,3 +332,33 @@ class SaslAuthenticateRequest_v0(Request):
 
 SaslAuthenticateRequest = [SaslAuthenticateRequest_v0]
 SaslAuthenticateResponse = [SaslAuthenticateResponse_v0]
+
+
+class CreatePartitionsResponse_v0(Response):
+    API_KEY = 37
+    API_VERSION = 0
+    SCHEMA = Schema(
+        ('topic_error_codes', Array(
```

Comment:
Seems like you missed `throttle_time_ms` field. Lets set the name to `topic_errors` to be exact to docs also. See https://kafka.apache.org/protocol#The_Messages_CreatePartitions.

Reply:
Ouch, my bad, pushed a fix.

Reply:
Is Line 335 in kafka/admin/client.py a residual mistake from this change?